qf/types.proto

```diff
@@ -55,7 +55,6 @@ message Group {
     uint64 ID          = 1;
     string name        = 2 [(go.field) = { tags: 'gorm:"uniqueIndex:group"' }];
     uint64 courseID    = 3 [(go.field) = { tags: 'gorm:"uniqueIndex:group"' }];
-    uint64 ScmTeamID   = 4;
```

Comment:
Will this be a problem for the database? Or can we just update the database table to ignore this attribute?